docs/src/main/asciidoc/grpc-getting-started.adoc

```diff
@@ -51,6 +51,17 @@ If you wish to generate code from different `proto` files for tests, also add th
 
 With this configuration, you can put your service and message definitions in the `src/main/proto` directory.
 `quarkus-maven-plugin` will generate Java files from your `proto` files.
+
+Please note that by default only the proto files in the `src/main/proto` and `src/test/proto` will be compiled and available as protobuf imports.
```

Comment:
Is there a reason for not having `quarkus.generate-code.grpc.scan-for-imports` as true by default ? Would it hurt the performance ?

Reply:
that's what I'm afraid. But let's see what others think

Reply:
Adding the proto files as imports doesn't seem to cost much, I'm afraid that scanning all the dependencies can cost a bit

Reply:
Yeah, not sure jandex has anything for resources. 
We may try to recognize well-known dependencies.

Reply:
I was actually thinking of giving users three options: 
 * `none` for the current behavior
 * `all` for scanning all dependencies
 * a comma separated list of groupId:artifactId to scan

With default of `com.google.protobuf:protobuf-java`, WDYT?

Reply:
the heavy lifting is done because I can pass the propetry through, I'll update the PR with ^

Reply:
Yes, that's what I meant. If `com.google.protobuf:protobuf-java` is there, or any well-known dependency containing protobuf files we should pick them (exception if configured otherwise such as a comma-separated list).

Reply:
done